packages/components/src/interfaces/complex/ImageGallery.ts

```diff
@@ -0,0 +1,36 @@
+import type { Static } from "@sinclair/typebox"
+import { Type } from "@sinclair/typebox"
+
+import type { IsomerSiteProps } from "~/types"
+import { AltTextSchema, ImageSrcSchema } from "./Image"
+
+export const IMAGE_GALLERY_TYPE = "imagegallery"
+
+const SingleImageSchema = Type.Object({
+  src: ImageSrcSchema,
+  alt: AltTextSchema,
+  caption: Type.Optional(Type.String({ maxLength: 100 })),
+})
```

Comment:
[comment, no actionable] i wonder if we should consolidate this together iwth our existing image schema to avoid an explosion of similar yet subtly different schemas

Reply:
thought about it - felt a bit hard since it's aldy not standardized at this moment, from schema shape to name of fields e.g. `src`, `imageSrc`